frontend/src/pages/vitrine/FAQ.jsx

```diff
@@ -1,81 +1,95 @@
+                    <Box sx={{ display:'flex', flexDirection:'column', justifyContent:'center',alignItems:'center'}}>
                         <h2 sx={{fontWeight: "bold"}}>Comment se rendre au Sixte ?</h2>
-                        <p sx={{}}>L’organisation ne prévoit pas de moyens de transports pour rejoindre le tournoi ; les participants doivent se rendre en voiture/ car sur place (Château de Montjoye, 78120 Clairefontaine-en-Yvelines) (parking disponible).</p>
+                        <Typography sx={{color: palette.secondary.main, marginBottom:'2rem'}}>L’organisation ne prévoit pas de moyens de transports pour rejoindre le tournoi ; les participants doivent se rendre en voiture/ car sur place (Château de Montjoye, 78120 Clairefontaine-en-Yvelines) (parking disponible).</Typography>
                     </Box>
-                    <Box sx={{padding:'2rem', display:'flex', flexDirection:'column', justifyContent:'center',alignItems:'center'}}>
+                    <Box sx={{ display:'flex', flexDirection:'column', justifyContent:'center',alignItems:'center'}}>
                         <h2 sx={{fontWeight: "bold", margin:'0.1'}}>Quel matériel prévoir ?</h2>
-                        <p sx={{}}>Le matériel que doivent prévoir les joueurs : chaussures à crampons moulés, les crampons vissés sont interdits (terrains synthétiques), protèges-tibias, ballons d’échauffement (ballons non prêtés)</p>
+                        <Typography sx={{color: palette.secondary.main, marginBottom:'2rem'}}>Le matériel que doivent prévoir les joueurs : chaussures à crampons moulés, les crampons vissés sont interdits (terrains synthétiques), protèges-tibias, ballons d’échauffement (ballons non prêtés)</Typography>
                     </Box>
-                    <Box sx={{padding:'2rem', display:'flex', flexDirection:'column', justifyContent:'center',alignItems:'center'}}>
+                    <Box sx={{ display:'flex', flexDirection:'column', justifyContent:'center',alignItems:'center'}}>
                         <h2 sx={{fontWeight: "bold", margin:'0.1'}}>Qu’est-ce que propose la buvette ?</h2>
-                        <p sx={{}}>La buvette propose des boissons chaudes et froides.</p>
+                        <Typography sx={{color: palette.secondary.main, marginBottom:'2rem'}}>La buvette propose des boissons chaudes et froides.</Typography>
                     </Box>
                 </Box>
-                <Box sx={{textAlign: 'left',paddingLeft:'4rem'}}>
-                    <p sx={{}}>Une autre question ? Envoie un mail à <a href="sixte@cs-sports.fr" style={{ textDecoration: 'none' }}>sixte@cs-sports.fr</a></p>
+                <Box sx={{textAlign: 'left', paddingTop:'2rem', paddingBottom:'2rem'}}>
+                    <Typography sx={{color: palette.secondary.main}}>Une autre question ? Envoie un mail à <a href="sixte@cs-sports.fr" style={{ textDecoration: 'none' }}>sixte@cs-sports.fr</a></Typography>
```

Comment:
```suggestion
                <Typography sx={{color: palette.secondary.main}}>
                    Une autre question ? Envoie un mail à 
                    <Link href="mailto:sixte@cs-sports.fr" sx={{ textDecoration: 'none' }}>
                        sixte@cs-sports.fr
                    </Link>
                </Typography>
```

https://mui.com/material-ui/react-link/

Globalement, un peu tous les trucs basiques de html css ça existe déjà en utilisant mui.
Faut vérifier que le changement marche bien, je suis pas sûr à 100% de la syntaxe mais dans l'idée c'est + clean de faire ça